hook.go

```diff
@@ -39,11 +39,13 @@ type ElasticHook struct {
 }
 
 type message struct {
-	Host      string        `json:"host"`
-	Timestamp string        `json:"@timestamp"`
-	Message   string        `json:"message"`
-	Data      logrus.Fields `json:"data"`
-	Level     string        `json:"level"`
+	Host      string `json:"host,omitempty"`
+	Timestamp string `json:"@timestamp"`
+	File      string `json:"file,omitempty"`
+	Func      string `json:"func,omitempty"`
+	Message   string `json:"message,omitempty"`
+	Data      logrus.Fields
+	Level     string `json:"level,omitempty"`
```

Comment:
What is the reason you made all the previously existed fields omitempty and removed the struct tag from Data?

Reply:
Hi, I cherry picked this fix from https://github.com/sohlich/elogrus/pull/29. I'm not the author.

I've noticed this fixes two issues:
1) `Data` field was not being sent to ES.  It is broken current.
2) `File` and `Func` are optional fields based on is caller reporting is enabled.
3) I'm not sure what the reasoning that Host and Level are now optional fields.  I'm guessing to allow a user to pass `nil` when creating a hook.  
e.g.
```
  hook, err := elogrus.NewAsyncElasticHook(
    client,
    host, // Can be nil
    config.LogLevel, // Can be nil
    config.LogSource,
  )
  ```

Reply:
I see, thanks for clarifications. Can you please at least add `json:"data,omitempty"` to the Data field (for consistency). And I will merge the PR.

Reply:
Done